cmd/ltagent/init.go

```diff
@@ -126,9 +129,98 @@ func RunInitCmdF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error while initializing loadtest: %w", err)
 	}
 
+	err = genAdmins(config, userPrefix)
+	if err != nil {
+		return fmt.Errorf("error while generating admin users: %w", err)
+	}
+	mlog.Info("admin generation completed")
+
 	return genData(lt, 50)
 }
 
+func genAdmins(config *loadtest.Config, userPrefix string) error {
+	mlog.Info(fmt.Sprintf("generating %d admins", config.InstanceConfiguration.NumAdmins))
+
+	transport := &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		DialContext: (&net.Dialer{
+			Timeout:   1 * time.Second,
+			KeepAlive: 30 * time.Second,
+		}).DialContext,
+		MaxConnsPerHost:       500,
+		MaxIdleConns:          500,
+		MaxIdleConnsPerHost:   500,
+		ResponseHeaderTimeout: 5 * time.Second,
+		IdleConnTimeout:       90 * time.Second,
+		TLSHandshakeTimeout:   1 * time.Second,
+		ExpectContinueTimeout: 1 * time.Second,
+	}
```

Comment:
I am thinking we could use the default transport for this. We are doing sequential requests anyway so it should work just fine.